lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -489,15 +489,13 @@ spec = do
             |] `shouldBe` (Left @String @(AddressAmount (ApiT Address, Proxy ('Testnet 0))) msg)
 
         it "ApiT PoolId" $ do
-            let msg = "Error in $: Invalid stake pool id: expecting a \
-                      \hex-encoded value that is 28 or 32 bytes in length."
+            let msg = "Error in $: Invalid stake pool id: expecting a Bech32 encoded value with human readable part of 'pool'."
```

Comment:
**Request**: break up overly-long lines (see https://github.com/input-output-hk/adrestia/wiki/Coding-Standards#limit-line-length-to-80-characters) with the line continuation character.

This helps with readability when reading two sources side-by-side.

```suggestion
            let msg = 
                    "Error in $: Invalid stake pool id: expecting a Bech32 \  
                    \encoded value with prefix 'pool'."
```